apps/sms/views/shim_host.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Messages Shim Host</title>
+    <!-- Firefox Desktop shims -->
+    <script defer src="/desktop-mock/navigator_moz_mobilemessage.js"></script>
+    <!-- Shim scripts -->
+    <script defer src="/lib/bridge/bridge.js"></script>
+    <script defer src="/lib/bridge/plugins/stream/service.js"></script>
+    <script defer src="/views/shared/js/utils.js"></script>
+    <script defer src="/services/js/moz_mobile_message/moz_mobile_message_shim.js"></script>
```

Comment:
Yeah, we'll need it, I haven't added it because moz_mobile_message_shim is not migrated to it yet in this PR :)